lib/ddtrace/tracer.rb

```diff
@@ -162,6 +162,8 @@ def configure(options = {})
     # Set the information about the given service. A valid example is:
     #
     #   tracer.set_service_info('web-application', 'rails', 'web')
+    #
+    # set_service_info is deprecated, no service information needs to be tracked
```

Comment:
should we make it already a no-op? since we're not sending it, maybe it's worth it just to keep the method/same signature but just ignore the call and drop it.

What do you think?

Reply:
yeah, I thought about that too, I need to trace through the code and see where `@services` is used. I'll do that now

Reply:
if it's used, just allocate the map but don't assign things to it.

Reply:
done